<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sign_in_window</class>
 <widget class="QMainWindow" name="sign_in_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/new/prefix2/smallized/sign_in copy.jpg);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>390</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
background-image: url(:/new/prefix2/smallized/tranc.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_usrnm">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>215</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
font: 700 24pt &quot;Roboto&quot;;
background-image: url(:/new/prefix2/smallized/tranc.png);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_psswrd">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>280</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
font: 700 24pt &quot;Roboto&quot;;
background-image: url(:/new/prefix2/smallized/tranc.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>360</y>
      <width>401</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
background-image: url(:/new/prefix2/smallized/tranc.png);
font: 700 9pt &quot;Roboto&quot;;</string>
    </property>
    <property name="text">
     <string>Access Denied</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
